packages/frontend/src/components/profile/remove_account/ui/PasswordConfirmationModal/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, useEffect, useState, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { currentTargetValue, forkEvent } from '../../../../../shared/lib/forms/selectors';
+import { isPasswordValid } from '../../../../accounts/EnterPasswordForm/lib/validate';
+import { inLength } from '../../../../accounts/SetPassword/lib/validation';
+import Modal from '../../../../common/modal/Modal';
+import PasswordInput from '../../../../common/PasswordInput';
+import { StyledButtons, StyledTitle } from './ui';
+
+type PasswordConfirmationModalActions = {
+    onClose: VoidFunction;
+    onConfirm: VoidFunction;
+};
+
+const PasswordConfirmationModal: FC<PasswordConfirmationModalActions> = ({
```

Comment:
Why created a new component instead use of EnterPasswordForm?

Reply:
<img width="718" alt="image" src="https://user-images.githubusercontent.com/4532214/200652664-06b2e151-e5d7-4508-91ba-98906039c7f1.png">

Reply:
I thought aboout it, but wasn't sure about condition changes.